actions/lib/formatters.py

```diff
@@ -4,8 +4,7 @@
 ]
 
 
-def to_issue_dict(issue, include_comments=False, include_attachments=False,
-                  include_customfields=False):
+def to_issue_dict(issue, include_comments=False, include_attachments=False, include_customfields=False, include_components=False, include_subtasks=False):
```

Comment:
@dylan-bitovi The CI failure is valid and related to changes made in this PR. Please check the lint error message: 
https://app.circleci.com/pipelines/github/StackStorm-Exchange/stackstorm-jira/37/workflows/9ec4706a-a690-4a1b-98e9-6dc3cdf6d6d9/jobs/395

```
actions/lib/formatters.py:7:101: E501 line too long (154 > 100 characters)
```